lib/processors/jsdoc/lib/ui5/plugin.js

```diff
@@ -1009,11 +1012,15 @@ function convertValueWithRaw(node, type, propertyName) {
 				raw: "{}"
 			};
 		}
-
+		
 	} else if ( isTemplateLiteralWithoutExpression(node) ) {
+		let value = {};
+		if (node && node.quasis && node.quasis[0] && node.quasis[0].value) {
+			value = node.quasis[0].value;
+		}
 		return {
-			value: node?.quasis?.[0]?.value?.cooked,
```

Comment:
I think here we can expect that `node.quasis[0].value` is defined because `isTemplateLiteralWithoutExpression` already checks for that.

Reply:
Thanks! Fixed it